superset-frontend/packages/superset-ui-core/src/color/colorSchemes/categorical/modernSunset.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import CategoricalScheme from '../../CategoricalScheme';
+import { ColorSchemeGroup } from '../../types';
+
+const schemes = [
+  {
+    id: 'modernSunset',
+    label: 'Modern sunset',
+    group: ColorSchemeGroup.Featured,
+    colors: [
```

Comment:
I'm not sure if these should go into the `theme` object. Could you add a `TODO` comment to revisit this file when working on [SIP-82 Improving Superset Theming](https://github.com/apache/superset/issues/20159)?

Reply:
👍

Reply:
Do you think all the other colour scheme files (including the old ones) should be revisited? I can add todos to all of them so that we don't miss them

Reply:
That would be great! Thank you!